### ISSUE TITLE
How to visualize the face mesh without texture and color?

### ISSUE THREAD

**Issue by YiwuZhong** [sicxu/Deep3DFaceRecon_pytorch#31]
Hi,

Thanks for your great work! I was wondering is there a way to visualize the face mesh on 2D image without texture and color?

One good example is 3rd column of Figure 2 in your paper. The default output is 2nd column in Figure 2 and no direct output can be the 3rd column visualization.

Best,
Yiwu

**Comment by haonanhe** [comment]
Hi, it also bothers me now. Have you found any solution? Thanks.

**Comment by chenhao-user** [comment]
hi, it is currently troubling me as well. Have you managed to solve it?
I made changes to these two lines of code, but I didn't get the correct result.
`def compute_color(self, face_texture, face_norm, gamma):
        gamma = torch.ones_like(gamma)
        .....
        face_color = torch.cat([r, g, b], dim=-1)
        .....
`

**Comment by StephanPan** [comment]
https://github.com/sicxu/Deep3DFaceRecon_pytorch/blob/b9e6fa89ea1566bb9ec27922f6f5d87c34b4b261/models/bfm.py#L294
face_texture = torch.ones(1,35709,3).float().cuda()*0.7  
you could set the face_texture to a pure color here
